app/static/styles/main.css

```diff
@@ -89,4 +89,64 @@ form {
   font-family: "Roboto", serif;
   font-weight: 700 !important;
   padding-top: 10px;
-}
\ No newline at end of file
+}
+
+h2 {
+  color: black;
+  font-size: 50px;
+  width: 40%;
+  padding-top: 120px;
+}
+
+h3 {
```

Comment:
this is the code that causes the Login and Sign up text's UI bug : ) because both LogIn and SignUp / the big text in home page use <h3>, and we're sharing one main.css file.